modules/programs/oh-my-posh.nix

```
@@ -52,6 +54,14 @@ in
       '';
     };
 
+    configFile = lib.mkOption {
+      type = lib.types.nullOr lib.types.path;
```

```suggestion
      type = with lib.types; nullOr (either str path);
```

Upstream seems to suggest this should support string or path. 

>The --config flag can accept either a local filepath or a remotely hosted config file.
>
>For example, the following is a valid --config flag: --config 'https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/jandedobbeleer.omp.json'